### ISSUE TITLE
Why is the validation set loss lower than the training set？

### ISSUE THREAD

**Issue by yonghua-wu** [lee-zq/VesselSeg-Pytorch#6]
Why is the validation set loss lower than the training set？
